plugin/pkg/project/validator/admission_test.go

```diff
@@ -70,12 +70,6 @@ var _ = Describe("Admission", func() {
 				Expect(admissionHandler.Admit(context.TODO(), attrs, nil)).To(Succeed())
 			})
 
-			It("should prevent creating the project because namespace prefix is missing", func() {
```

Comment:
The rest of the tests for the Project namespace could be also removed. There is no validation code in the admission plugin that validates the `.spec.namespace` field.

Reply:
I pushed a33f64e79a3c6c5d3ce981804ec357cfd2281278 to address this PR review comment.